buildSrc/src/main/kotlin/com/datadog/gradle/config/AndroidConfig.kt

```diff
@@ -17,7 +17,7 @@ object AndroidConfig {
     const val MIN_SDK_FOR_COMPOSE = 21
     const val BUILD_TOOLS_VERSION = "31.0.0"
 
-    val VERSION = Version(1, 15, 0, Version.Type.Dev)
+    val VERSION = Version(1, 15, 0, Version.Type.Snapshot)
```

Comment:
is it enough to publish the snapshot on Sonatype? will it skip staging/approval step if version has `-SNAPSHOT` suffix? Or will it be some other PR?

Reply:
Sonatype will automatically skip staging based on the `-SNAPSHOT` suffix